specs/order_spec.rb

```diff
@@ -79,32 +80,40 @@
 end
 
 # TODO: change 'xdescribe' to 'describe' to run these tests
-xdescribe "Order Wave 2" do
+describe "Order Wave 2" do
   describe "Order.all" do
     it "Returns an array of all orders" do
-      # TODO: Your test code here!
+      result = Grocery::Order.all
+      result.must_be_instance_of(Array)
     end
 
     it "Returns accurate information about the first order" do
-      # TODO: Your test code here!
+      first_order = Grocery::Order.find(1)
+      first_order.products.must_include("Slivered Almonds")
```

Comment:
This is a test for `Order.all`, but you don't actually call `.all` here. If you replaced line 91 with `first_order = Grocery::Order.all.first` this test would be good.

Remember, the _Act_ step of _Arrange_, _Act_, _Assert_ is always to call the method you're testing.